### PR TITLE
feature(proctree): enrich events with executable from proctree

### PR DESCRIPTION
**Pull request by rafaeldtinoco**
Rebase of https://github.com/aquasecurity/tracee/pull/3511

### CODE REVIEW

File: pkg/ebpf/processor_proctree.go [hunk 1]
```diff
@@ -243,3 +245,38 @@ func (t *Tracee) procTreeExitProcessor(event *trace.Event) error {
...
+				"event name", event.EventName,
+				"timestamp", event.Timestamp,
+				"error", "not found in process tree",
+			)
+		}
+		return nil // threads don't have executable info in the process tree
+	}
+
+	// Event timestamp is changed to relative (or not) at the end of all processors only.
+	eventTimestamp := time.Unix(0, int64(event.Timestamp)+int64(t.bootTime))
```
Comment (yanivagman) [inline pkg/ebpf/processor_proctree.go:272]: How is this related to this PR?
Also - isn't it already taken care of by normalize time processor?

**Review by yanivagman — COMMENTED** [review]
After this is merged, there should be a follow up PR to add executable to the new event structure and initialize it @josedonizetti

File: pkg/ebpf/processor_proctree.go (hunk 1 above)
Comment (rafaeldtinoco) [reply pkg/ebpf/processor_proctree.go:272]: Event timestamp is ns since boot and proctree timestamp is ns since epoch. The time normalization (that will convert time to be epoch or not) runs at the end of all processors (so it does not disturb what processors expect).

**Comment by rafaeldtinoco** [comment]
All the errors on tests were related to timeouts (in the very last step of the tests).

**Comment by rafaeldtinoco** [comment]
> After this is merged, there should be a follow up PR to add executable to the new event structure and initialize it @josedonizetti

I think that has been already taken care of:

image

If not we can "fix" it after the branch out (since it is happening today).

**Comment by yanivagman** [comment]
> > After this is merged, there should be a follow up PR to add executable to the new event structure and initialize it @josedonizetti
> 
> I think that has been already taken care of:
> 
> image
> 
> If not we can "fix" it after the branch out (since it is happening today).

Should also have "path" in it, which is missing, and the value needs to be initialized
